snuba/datasets/profiles.py

```diff
@@ -5,3 +5,7 @@
 class ProfilesDataset(Dataset):
     def __init__(self) -> None:
         super().__init__(default_entity=EntityKey.PROFILES)
+
+    @classmethod
+    def is_experimental(cls) -> bool:
+        return True
```

Comment:
The PR description states the experimental dataset functionality is not used yet. It seems to be applied here though.

Reply:
will remove